tests/bootstrap.php

```diff
@@ -6,6 +6,7 @@
  */
 
 \define( 'ACTIVITYPUB_DISABLE_INCOMING_INTERACTIONS', false );
+\define( 'WP_ENVIRONMENT_TYPE', 'staging' );
```

Comment:
```
PHP Warning:  Constant WP_ENVIRONMENT_TYPE already defined in /wordpress-phpunit/wp-tests-config.php on line 118
```
I think we need to define it in `.wp-env.json` to override the default constant. When testing that, you'll need to stop and then start wp-env again, for the constant to be set correctly.

Reply:
```
% npm run env-test

> env-test
> wp-env run tests-cli --env-cwd="wp-content/plugins/activitypub" vendor/bin/phpunit

ℹ Starting 'vendor/bin/phpunit' on the tests-cli container. 

[29-Jul-2025 12:43:50 UTC] PHP Warning:  Constant WP_ENVIRONMENT_TYPE already defined in /wordpress-phpunit/wp-tests-config.php on line 118
Installing...
Running as single site... To run multisite, use -c tests/phpunit/multisite.xml
Not running ajax tests. To execute these, use --group ajax.
Not running ms-files tests. To execute these, use --group ms-files.
Not running external-http tests. To execute these, use --group external-http.
PHPUnit 8.5.42 by Sebastian Bergmann and contributors.

```

Reply:
I don't see it in phpunit GitHub action output. Either way, I think it should be in `.wp-env.json` to replace the default value for it.